src/backend/common/helpers/playoff_advancement_helper.py

```diff
@@ -654,34 +654,16 @@ def _alliance_name(
     ) -> Optional[str]:
         if not alliance_selections:
             return None
-        for (
-            alliance_selection
-        ) in alliance_selections:  # search for alliance. could be more efficient
+        for (n, alliance_selection) in enumerate(
+            alliance_selections
+        ):  # search for alliance. could be more efficient
             picks = alliance_selection["picks"]
             if (
                 len(set(picks).intersection(set(team_keys))) >= 2
             ):  # if >= 2 teams are the same, then the alliance is the same
-                return alliance_selection.get("name")
+                return alliance_selection.get("name", f"Alliance {n+1}")
```

Comment:
Technically the "name" field in the alliance module is optional, so we can add a fallback for when it is not found (because in Cheesy Arean's case, it doesn't populate alliance names)

https://github.com/the-blue-alliance/the-blue-alliance/blob/48ed1c9560b9adb1b2b9efab7464bb76b8d9012e/src/backend/common/models/alliance.py#L9-L16